litmus-portal/frontend/src/components/Header/styles.ts

```diff
@@ -6,13 +6,173 @@ const useStyles = makeStyles((theme: Theme) => ({
     height: '100%',
     backgroundColor: theme.palette.background.default,
   },
-  breadCrumbsGrid: {
-    marginTop: theme.spacing(2),
-    paddingLeft: theme.spacing(4),
+
+  headerFlex: {
+    height: '5%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: theme.palette.background.default,
+    marginTop: theme.spacing(1),
+    marginBottom: theme.spacing(0.5),
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(0.5),
+  },
+
+  headerFlexProfile: {
+    height: '5%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: theme.palette.background.default,
+    marginTop: theme.spacing(0),
+    marginBottom: theme.spacing(0),
+    paddingTop: theme.spacing(0),
+    paddingBottom: theme.spacing(0),
+    marginLeft: theme.spacing(5),
+    paddingleft: theme.spacing(5),
+  },
+
+  headerFlexPadded: {
+    height: '5%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: theme.palette.background.default,
+    marginTop: theme.spacing(0),
+    marginBottom: theme.spacing(0),
+    paddingTop: theme.spacing(1),
+    paddingBottom: theme.spacing(0),
+  },
+
+  headerFlexExtraPadded: {
+    height: '5%',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: theme.palette.background.default,
+    marginTop: theme.spacing(0.5),
+    marginBottom: theme.spacing(0),
+    paddingTop: theme.spacing(2),
+    paddingBottom: theme.spacing(0),
   },
-  notificationSection: {
+
+  avatarBackground: {
```

Comment:
Please add comments for which section the styles belong to

Reply:
sure.